libs/fetcher.client.js

```diff
@@ -171,6 +182,18 @@ function executeRequest (request, resolve, reject) {
             // TODO: Add test for this fallback
             get_uri = defaultConstructGetUri.call(request, uri, request.resource, request._params, clientConfig, request.options.context);
         }
+        // TODO: Remove `returnMeta` feature flag after next release
+        // This feature flag will enable the new return format for GET api requests
+        // Whereas before any data from services was returned as is. We now return
+        // an object with a data key containing the service response, and a meta key
+        // containing the service's metadata response (i.e headers and statusCode).
+        // We need this feature flag to be truly backwards compatible because it is
+        // concievable that some active browser sessions could have the old version of
+        // client fetcher while the server upgrades to the new version. This could be
+        // easily fixed by refreshing the browser, but the feature flag will ensure
+        // old fetcher clients will receive the old format and the new client will
+        // receive the new format
+        get_uri = get_uri + '&returnMeta=true';
```

Comment:
Is it guaranteed that `?` and one other query param will always be present? Might be safer to add this to `request._params` before calling `defaultConstructGetUri`.